db/migrate/20180712122000_remove_host_provisioning.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveHostProvisioning < ActiveRecord::Migration[5.0]
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Delete Host Provision Requests and Tasks") do
+      MiqRequest.where(:type => :MiqHostProvisionRequest).destroy_all
+      MiqRequestTask.where(:type => :MiqHostProvision).destroy_all
```

Comment:
Are there any relationships that need to be cleaned up as well?

Reply:
After reviewing the `MiqRequest` model I added `miq_approvals`.  I believe that is everything.